coordinator/src/test/scala/filodb.coordinator/queryplanner/LogicalPlanParserSpec.scala

```diff
@@ -148,6 +148,51 @@ class LogicalPlanParserSpec extends AnyFunSpec with Matchers {
     res shouldEqual "sum(rate(http_requests_total{job=\"app\"}[300s:60s] @100))"
   }
 
+  it("should correctly generate queries when LogicalPlan contains scalar() function") {
```

Comment:
can you add tests for query without any binary operators?

I mean no operators like `+-*/><`

for example
```
scalar(avg_over_time(http_requests_total[1h]))
```

Reply:
Understood.
I think binary operators are rarely used but could we keep other mathematical operators like `+-*/`?

Reply:
I've removed binary operators like `<>`

Reply:
oh < and > are valid operators.
I mean you had test cases for with binary operators.
And you also need to test the one without binary operators.

Since you added those. It looks good to me.